src/views/registration-js/attendees/fields/birth.php

```diff
@@ -6,13 +6,17 @@
  * Override this template in your own theme by creating a file at:
  * [your-theme]/tribe/tickets/registration-js/attendees/fields/birth.php
  *
- * @since   4.12.1
+ * @version TBD
+ *
+ * @since 4.12.1 Introduced template.
+ * @since TBD Fix PHP notice for undefined variable.
  *
  * @see     Tribe__Tickets_Plus__Meta__Field__Birth
  */
 
-$required    = isset( $field->required ) && 'on' === $field->required ? true : false;
+$required    = isset( $field->required ) && 'on' === $field->required;
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 2 spaces but found 4 spaces

<sub>hash: c4cfe3f588068a6ad31c641ea85f94d3</sub>